apps/src/lib/config/ethereum_bridge/params.rs

```diff
@@ -47,11 +49,11 @@ pub struct Contracts {
 /// upgraded. Starts from 1.
 #[derive(Clone, Copy, Eq, PartialEq, Debug, Deserialize, Serialize)]
 #[repr(transparent)]
-pub struct ContractVersion(u64);
+pub struct ContractVersion(NonZeroU64);
 
 impl Default for ContractVersion {
     fn default() -> Self {
-        Self(1)
+        Self(unsafe { NonZeroU64::new_unchecked(1) })
```

Comment:
maybe add a `SAFETY` comment here. it might be somewhat obvious why it's needed, but I think it's good practice to accompany `unsafe{}` blocks with `SAFETY` comments. for instance:
```rs
        // SAFETY: The only way the API contract of `NonZeroU64`can be violated is if we construct values of this type using 0 as argument.
        Self(unsafe { NonZeroU64::new_unchecked(1) })
```

Reply:
agreed, have added the comment